lcoe_calculator/tech_processors.py

```diff
@@ -563,7 +566,6 @@ class UtilityBatteryProc(AbstractBatteryProc):
     sheet_name = 'Utility-Scale Battery Storage'
     num_tds = 5
 
-
 class CommBatteryProc(AbstractBatteryProc):
```

Comment:
Very very minor: two blank lines between classes is standard Python style https://peps.python.org/pep-0008/#blank-lines